Formula/crc.rb

```diff
@@ -0,0 +1,26 @@
+class Crc < Formula
+  desc "OpenShift 4 cluster on your local machine"
+  homepage "https://code-ready.github.io/crc/"
+  url "https://github.com/code-ready/crc.git",
+      :tag      => "1.0.0",
+      :revision => "575079bff638214dc46d831caaef1c08927346ec"
+  head "https://github.com/code-ready/crc.git"
+
+  depends_on "go" => :build
+
+  def install
+    # Determine install OS and architecture
+    os = `go env GOOS`.strip
+    arch = `go env GOARCH`.strip
+    os = "macos" if os == "darwin"
+
+    # Make and install binary
+    system "make", "out/#{os}-#{arch}/crc"
```

Comment:
Simply hardcode values…
```suggestion
    system "make", "out/darwin-amd64/crc"
```
That's what we generally do…

Reply:
I did this so that this formula is linuxbrew compatible out of the box.

Reply:
I see the appeal, but we should be consistent and do it throughout if we are to do it. Right now, we write formulas for Darwin.